conf/server/server_full.conf

```diff
@@ -100,6 +100,21 @@ server {
     # jwt_issuer = ""
 
     # log_file: File to write logs to
+    #
+    # If set, spire-server will spawn a handler to reopen the file upon receipt
+    # of SIGUSR2 to support log rotation. To use logrotate without lossy
+    # copytruncate option, users MUST add a postrotate script to the logrotate
+    # configuration to send the SIGUSR2 signal to the spire-server process.
+    #
+    # Minimal example of logrotate.conf:
+    #
+    # /path/to/spire-agent.log {
+    #     rotate 7
+    #     postrotate
+    #         killall -USR2 spire-server
```

Comment:
example for server file has `killall spire-server` but log file is `path/to/spire-agent.log`

Reply:
Thank you.